--- docs-source/DoxyfileC++.in	2018-03-20 21:11:05.000000000 -0400
+++ docs-source/DoxyfileC++-modified.in	2018-03-20 21:11:42.000000000 -0400
@@ -68,7 +68,7 @@
 #---------------------------------------------------------------------------
 # configuration options related to warning and progress messages
 #---------------------------------------------------------------------------
-QUIET                  = NO
+QUIET                  = YES
 WARNINGS               = YES
 WARN_IF_UNDOCUMENTED   = YES
 WARN_IF_DOC_ERROR      = YES
